src/styles/Search.css

```diff
@@ -0,0 +1,27 @@
+.search-form {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.search-form__input {
+  width: 20%;
+  height: 50px;
+  border-radius: 11px;
+  font-size: 20px;
+  padding: 0 10px;
+}
+
+.search-form__button {
+  margin: 10px 0 0 10px;
```

Comment:
Using margin like this is a bit odd... Am I right in thinking you're trying to centre the button vertically? If so you could declare `margin-right: 10px` on this element, and add `align-items: center` to the `.search-form` class which would be much tidier